### ISSUE TITLE
families presented in a 'natural order' within chapter/level

### ISSUE THREAD

**Issue by ottosnotebook** [eruditionapp/eruditionapp#66]
tl;dr version: it's important to have the content creators specify the order in which cards will be presented for the first time (after that, spaced repetition takes over) for reasons specified below. This could be accomplished by having the CCs specify order using an index (e.g., placing the numbers 1 through n next to their questions), or order them in the excel doc. any other ideas?

longer version:
from pseudocode, updated 5-2-16: 

"All cards in the database for a specific title will have a natural order (that we can sometimes supersede with some randomization commands described later). If no dependency or randomization commands are put in place to supersede the natural order, then the cards simply move from pending to new in the natural order. All cards therefore have a field that is an order number (=1 to 9999), which is different from their unique ID numbers." 

the purpose of having a "natural order" of families within levels is so the content creators can specify the exact order in which users will see families/questions. it's also an imperative feature if the level locking/unlocking mechanisms are going to work property. here's an example:

say a user is working their way through the deck on Marx' capital, just having unlocked level 3: chapter level factual detail for chapter 1 of the book. the level unlocking mechanism specified in the pseudocode allows the user to answer 10% of cards at chap. 1, level 3 before it unlocks the next level: ch 1, level 4. because the first chapter of capital is so rich with important excerpts, interpretations, commentaries, etc., let's say we have 100 families just for chapter 1, level 3. this means the user could unlock the next level in as little as 10 cards (10% of 100 cards). and if we're having 4 levels max unlocked at the same time, the user could progress to level 7 (at this point, the user has levels 4-7 unlocked, with level 3 locked) without having seen all 100 cards from that level.

("I do have a thesis") - it would be helpful for the content creators to have the ability to mandate that a user sees certain cards first, before the user locks them by progressing further. this way, they can place a representative sample of, say, 40 cards _first_ in the excel doc to make sure that ch. 1 level 3 will not lock until a user has seen those 40 (of 100 total) cards. let me know if the example/my explanation isn't clear
